virttest/qemu_vm.py

```diff
@@ -1183,7 +1183,7 @@ def add_cpu_flags(devices, cpu_model, flags=None, vendor_id=None,
                 cmd += ",family=%s" % family
             return cmd
 
-        def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
+        def add_boot(devices, opts):
```

Comment:
I guess,  this line should be 

def add_boot(devices , **opts): 

right?

Reply:
@xutian 
Frist def add_boot(devices, opts):  works.   Before, I try to use "**opts" pass dict type. But it shows  like this error message "takes exactly 1 argument (2 given)".